include/xtensor/xtensor_config.hpp

```diff
@@ -11,8 +11,8 @@
 #define XTENSOR_CONFIG_HPP
 
 #define XTENSOR_VERSION_MAJOR 0
-#define XTENSOR_VERSION_MINOR 21
```

Comment:
I think this change is not backward incompatible so the version should be 0.21.5-dev.

Reply:
Fixed.